src/Mono.Android.Export/Mono.Android.Export.csproj

```diff
@@ -1,42 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="..\..\Configuration.props" />
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{B8105878-D423-4159-A3E7-028298281EC6}</ProjectGuid>
-    <ProjectTypeGuids>{EFBA0AD7-5A72-4C68-AF49-83D382785DCF};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <OutputType>Library</OutputType>
-    <RootNamespace>Mono.Android.Export</RootNamespace>
-    <AssemblyName>Mono.Android.Export</AssemblyName>
+    <TargetFrameworks>monoandroid10;netcoreapp3.1</TargetFrameworks>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\product.snk</AssemblyOriginatorKeyFile>
+    <NoStdLib>true</NoStdLib>
+    <ImplicitlyExpandDesignTimeFacades>false</ImplicitlyExpandDesignTimeFacades>
   </PropertyGroup>
-  <Import Project="..\..\Configuration.props" />
-  <PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'monoandroid10' ">
     <TargetFrameworkIdentifier>MonoAndroid</TargetFrameworkIdentifier>
-    <TargetFrameworkVersion>$(AndroidFrameworkVersion)</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v1.0</TargetFrameworkVersion>
```

Comment:
@jonpryor I am missing some historical context here, do we need to be compiling against a specific MonoAndroid TargetFrameworkVersion here, or do you think these changes will be compatible?

Reply:
I would have thought that this *wouldn't* be compatible -- because I would have thought that we needed to use the "same" `$(TargetFrameworkVersion)` as `Mono.Android.dll` was built with so that it could be found.  Except there's a `@(ProjectReference)` to `Mono.Android.csproj`, which presumably means we *don't* need to use the same `$(TargetFrameworkVersion)` value!

*Furthermore* this change might solve the issues I was having on #4376!